incubator/react-native-error-trace-decorator/src/index.ts

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+import { symbolicateStackTraces } from "./symbolicate_stack_traces";
+
+symbolicateStackTraces();
```

Comment:
Can we rename this to a `cli.ts`, and make `index.ts` re-export the function instead:

```suggestion
export { symbolicateStackTraces } from "./symbolicate_stack_traces";
```

Reply:
That makes sense. Sure I will do accordingly

Reply:
Done